arbos/arbosState/arbosstate.go

```diff
@@ -332,6 +331,9 @@ func (state *ArbosState) UpgradeArbosVersion(
 			ensure(params.UpgradeToVersion(2))
 			ensure(params.Save())
 
+		case 32:
+			// no change state needed
```

Comment:
This needs to return an error if it's not a debug chain, otherwise out-of-date nodes will process the upgrade. We could just generally error if the version we're upgrading to is less than MaxArbosVersionSupported. That might be the way to go.